Ginger/GingerCore/Actions/ActSwitchWindow.cs

```diff
@@ -100,7 +100,7 @@ public Int32 WaitTime
 
         bool IObsoleteAction.IsObsoleteForPlatform(ePlatformType platform)
         {
-            if (platform == ePlatformType.Web || platform == ePlatformType.Mobile || platform == ePlatformType.NA)
+            if (platform == ePlatformType.Web || platform == ePlatformType.NA)
             {
```

Comment:
i asked to remove the Mobile only for GenericElement Action and leave it for rest